packages/flutter_tools/pubspec.yaml

```diff
@@ -34,6 +34,7 @@ dependencies:
   native_stack_traces: 0.3.7
   build_daemon: 2.1.4
   shelf: 0.7.5
+  vm_snapshot_analysis: ^0.5.0+1
```

Comment:
this should be an exact version, then run `flutter update-packages --force-upgrade` to update the checksums